lib/cloud/watchers/watcher.go

```diff
@@ -107,16 +133,58 @@ func (w *Watcher) Start() {
 		case <-ticker.C:
 			w.fetchAndSend()
 		case <-w.ctx.Done():
-			w.log.Debugf("Cloud databases watcher done.")
+			w.log.Debugf("Cloud databases and ec2 watcher done.")
 			return
 		}
 	}
 }
 
+func (w *Watcher) startEC2() {
```

Comment:
```suggestion
func (w *Watcher) runEC2() {
```

Typically "start" is used for things that kick off some background processing and then return, where "run" is preferred for long running operations that block.